src/main.h

```diff
@@ -211,7 +215,7 @@ std::string GetWarnings(std::string strFor);
 /** Retrieve a transaction (from memory pool, or from disk, if possible) */
 bool GetTransaction(const uint256 &hash, CTransaction &tx, uint256 &hashBlock, bool fAllowSlow = false);
 /** Find the best known block, and make it the tip of the block chain */
-bool ActivateBestChain(CValidationState &state, CBlock *pblock = NULL);
+bool ActivateBestChain(CValidationState &state, CBlock *pblock = NULL, libzcash::ProofVerifier& verifier = disabledVerifier);
```

Comment:
I intentionally made it have a default value - as several methods are calling the methods that now have this verifier parameter, and didn't want all of them to need to define a disabled verifier and pass it